pkg/server/federation/bundle.go

```diff
@@ -0,0 +1,76 @@
+package federation
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"time"
+
+	"github.com/spiffe/spire/pkg/common/bundleutil"
+	"gopkg.in/square/go-jose.v2"
+)
+
+type bundleDoc struct {
+	jose.JSONWebKeySet
+	Sequence    uint64 `json:"spiffe_sequence,omitempty"`
+	RefreshHint int64  `json:"spiffe_refresh_hint,omitempty"`
```

Comment:
curious why RefreshHint is not also uint64

Reply:
I usually prefer signed integers over unsigned unless they are spec'd that way or i need something above the 2^63 range. Considering RefreshHint is supposed to be a (relatively) small integer value number of seconds, i might just use `int`. Thoughts?

Reply:
It's no biggie, just stuck out to me since it's defined next to a uint. Made me wonder if you expected it to be negative at any point